pinot-common/src/main/java/org/apache/pinot/common/response/broker/BrokerResponseStats.java

```diff
@@ -49,7 +50,7 @@ public class BrokerResponseStats extends BrokerResponseNative {
   private long _stageExecutionTimeMs = 0;
   private int _stageExecutionUnit = 0;
   private long _stageExecWallTimeMs = -1;
-  private Map<String, Map<String, String>> _operatorStats = new HashMap<>();
+  private Map<OperatorId, Map<String, String>> _operatorStats = new HashMap<>();
```

Comment:
I dont really think it is necessary to change the String type to OperatorId type -->
what we really need is to ensure that the same `requestId + stageId + operatorId` --> operatorId is monotonic increasing from the bottom receive operator to the top send operator. the actual JSON doesn't need to be sorted --> we can always post process or do the sorting when rendering the JSON

Reply:
yes, ensuring operatorId is monotonically increasing is done by operatorIndex. I can revert it back to string.